Add `The Followpoint: xootynator, the CEO of Full Area` news post (#12103)
ppy/osu-wiki: news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

---
layout: post

> title: "The Followpoint: xootynator, the CEO of Full Area"

Reviewer: was told that letting the interviewees choose their own news title doesn't seem right

maybe
```suggestion
title: "The Followpoint: xootynator, the Full Area Appreciator"
```

though i wouldn't particularly mind
```suggestion
title: "The Followpoint: xootynator, the Full Area Stan"
```

Author: Figured I'd repeat here what I said on Discord. If there is no actual issue with the title excluding the fact it was suggested by xooty (actually not even her, but her moderator, she just told me) - I am against changing it, especially to something worse. I liked the title myself, as well as Utiba who I discussed it with, and felt like it's a better one to use that original, so I don't see a problem with keeping it.

Reviewer 2: Revert to original IMO.

Author: @peppy sure, but may I ask why? Is the reason really just that it wasn't my idea?

Reviewer 2: > was told that letting the interviewees choose their own news title doesn't seem right

Author: So it really is the only reason, understood

date: 2024-09-16 16:00:00 +0000
series: The Followpoint
---

Get ready for an artistic spin with xootynator, the sorceress of full area!

![](/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/banner.jpg)

> Guess what? The Followpoint is back to bring you another banger, an interview with [xootynator](https://osu.ppy.sh/users/3717598)! You likely heard of her at least once because of her community contributions (such as tournament staffing or content making), tournament participation, arts or her unique playstyle. And today we have a chance to learn more about her and her osu! journey.

Reviewer 3: i might be completely in the wrong here but are you sure that she's known for tournament staffing? afaik she's a really active player but not staffer

Reviewer 3: also, i dont think there's a need for a bracket here if you are already listing everything
```suggestion
Guess what? The Followpoint is back to bring you another banger, an interview with [xootynator](https://osu.ppy.sh/users/3717598)! You likely heard of her at least once because of her community contributions such as tournament staffing, content making, tournament participation, arts or her unique playstyle. Today we have a chance to learn more about her and her osu! journey.
```

![](/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/oatmeal_and_daffy.jpg)

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

How did you get into drawing? Was there anything that motivated you to learn it?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> I got into art when I was very young, always doodling on my homework and such. I first got a tablet when I was 12 years old and was pretty into my DeviantArt and Instagram phase doing a lot of My Little Pony stuff... but I think what inspired me the most as a child was looking at old concept art for games that me and my cousins would play. I have a soft spot in my heart for the concept work of the game Scaler, which featured fantasy landscapes and creatures!

Reviewer 3: ```suggestion
I got into art when I was very young, always doodling on my homework and such. I first got a tablet when I was 12 years old and was pretty into my DeviantArt and Instagram phase doing a lot of My Little Pony stuff... but I think what inspired me the most as a child was looking at old concept art for games that my cousins and I would play. I have a soft spot in my heart for the concept work of the game Scaler, which featured fantasy landscapes and creatures!
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

How did you get introduced to osu! and what was your first impression of it?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> My first introduction to osu! was actually in grade 4. I have a vivid memory of someone playing [Osu! Tatakae! Ouendan](/wiki/FAQ#i've-never-played-this-"ouendan"-thing-—-what-is-it) on their DS outside the gym, and I think there were some visuals of a guy trying to impress a girl on a date, and if you missed any notes then the guy would turn into a werewolf...

Reviewer 3: it would be really cool if you could find this level and post a screenshot of it in between the two paragraphs

> Eventually when I started highschool, I saw people playing osu! on the school computers. It felt like some spiritual sign that I should play it since I always had that fond memory of that time by the gym. So at some point I tried it at home and the rest is prety much history. I don't remember much of my first days of the game but I knew I was hooked right away. I still play it to this day after all!

Reviewer 3: ```suggestion
Eventually when I started highschool, I saw people playing osu! on the school computers. It felt like some spiritual sign that I should play it since I always had that fond memory of that time by the gym. So at some point I tried it at home and the rest is pretty much history. I don't remember much of my first days of the game but I knew I was hooked right away. I still play it to this day after all!
```

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

Not at all, I'm pretty sure I was bootynator for about 2 days until having a shower thought that "xooty" would look much cooler than "booty" 😼

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Speaking of nicknames, what made you change your username to "his kitten" for some time?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> That was from my first subathon in 2023! I had a subscriber goal to change my name to anything chat chose, and they ended up choosing "his kitten". Everyone wanted me to change it back to xootynator after a while but I maybe got a little attached to the name... meow :3

Reviewer 3: ```suggestion
That was from my first subathon in 2023! I had a subscriber goal to change my name to anything chat chose, and they ended up choosing "his kitten". Everyone wanted me to change it back to xootynator after a while, but I maybe got a little attached to the name... meow :3
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

> I remember you mentioning it in one of your previous interviews, but can you tell us what are SFA (Stan Full Area) origins? And how did that become your thing?

Reviewer: ```suggestion
I remember you mentioning it in one of your previous interviews, but can you tell us what's the origin of SFA (Stan Full Area)? And how did that become your thing?
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> Stan Full Area has a pretty funny origin, since it was actually out team name for the 2020 iteration of [MRC (Mahloola Regional Cup)](https://osu.ppy.sh/community/forums/topics/1136787). The team was me, [Rupertion](https://osu.ppy.sh/users/5774823) and [Eddie-](https://osu.ppy.sh/users/3898396), three players who played with the entire area of our tablets. No changes to the size of the active area at all. Eddie is a big kpop fan, so gave us the name Stan Full Area. Since then it's sort of just stuck as a tag to recognize other people with the same playstyle!

Reviewer 3: ```suggestion
Stan Full Area has a pretty funny origin, since it was actually out team name for the 2020 iteration of [MRC (Mahloola Regional Cup)](https://osu.ppy.sh/community/forums/topics/1136787). The team was me, [Rupertion](https://osu.ppy.sh/users/5774823) and [Eddie-](https://osu.ppy.sh/users/3898396), three players who played with the entire area of our tablets. No changes to the size of the active area at all. Eddie is a big k-pop fan, so gave us the name Stan Full Area. Since then, it's sort of just stuck as a tag to recognize other people with the same play style!
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

What would you call the greatest gift that osu! has given you?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> osu! has given me so many opportunities in my life that it's crazy when I look back on it. Not only do I have the opportunity to travel and meet people around the world, have fun and make a career out of something I love AND feel like I have the drive to better my content and my skills, osu! also gave me the greatest gift of being able to connect with people. 

Reviewer 3: ```suggestion
osu! has given me so many opportunities in my life that it's crazy when I look back on it. Not only do I have the opportunity to travel and meet people around the world, have fun and make a career out of something I love AND feel like I have the drive to better my content and my skills, osu! also gave me the greatest gift of being able to connect with people.
```

> I'm not very social and I keep to myself most of the time, but I've made long lasting friendships during my entire time on osu!. I feel like I've gained a family. The community can seem toxic at times online, but from my experience, it's never the reality when you get to know people. Not only that, but building my own little community has also allowed people I know to make great connections and friendships with other people. When I take a step back, I feel like a little spider that's woven a big web of connections to all different kinds of aspects of my life. Without osu! I would be very different!

Reviewer 3: ```suggestion
I'm not very social, and I keep to myself most of the time, but I've made long-lasting friendships during my entire time on osu!. I feel like I've gained a family. The community can seem toxic at times online, but from my experience, it's never the reality when you get to know people. Not only that, but building my own little community has also allowed people I know to make great connections and friendships with other people. When I take a step back, I feel like a little spider that's woven a big web of connections to all different kinds of aspects of my life. Without osu! I would be very different!
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Did streaming push you into becoming a top player, or was it vice versa?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> I think it kind of goes both ways, but I totally started streaming and doing content creation because of the drive to become a top player. I have practically never had an osu! session off stream since I got unbanned in 2018 (that's a whole other story). I started posting on YouTube shortly after my first OWC appearance in late 2020, which really kickstarted my career. I wanted to do something every week that would get people interested. Be that FCing every map in my first match, or carrying my team to victory by taking extra mods on FreeMod picks. That ended up giving me a lot of traction on Twitch, and I used that as leverage to keep pushing my competitive skill and get better!

Reviewer: note that we usually never mention the topic of restrictions in news posts

Author: I understand that, but let's be honest, it really is an important part of many players' history

Reviewer 3: personally id leave it but lets see if ppy has anything against it

Author: > it's fine, although leaving out the reason is weird. did they cheat?

She was falsely banned like many players in the past, like WhiteCat if that changes things

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

Who would you want to meet IRL the most if you could choose any player, and why?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> The first person that comes to mind is probably [val0108](https://osu.ppy.sh/users/243917). I love pretty much all of their maps, and I used to always love the little Twitter updates they'd post about their life! At one point val0108 came in to my Twitch chat and I was star struck. I remember them saying something about my scores being cool with full area and it was inspiring, and I just think that val has a silly personality!

Reviewer 3: ```suggestion
The first person that comes to mind is probably [val0108](https://osu.ppy.sh/users/243917). I love pretty much all of their maps, and I used to always love the little Twitter updates they'd post about their life! At one point val0108 came in to my Twitch chat, and I was star struck. I remember them saying something about my scores being cool with full area, it was inspiring, and I just think that val has a silly personality!
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

How do you manage nerves/shake when setting leaderboard scores?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> My best piece of advice for nerves is always this: everyone gets nerves, but what makes you into a really good player is knowing how to play through those nerves. I personally don't shake much but my breathing gets shallow and I notice my mind starts to wander and worry about missing. What keeps me grounded is repeating thoughts like "I could miss, or I could also just FC" or to remind myself to stay focused on the main mechanics of my body. Don't lose your reading and listening, and keep your hands as loose as possible.

Reviewer 3: ```suggestion
My best piece of advice for nerves is always this: everyone gets nerves, but what makes you into a really good player is knowing how to play through those nerves. I personally don't shake much, but my breathing gets shallow, and I notice my mind starts to wander and worry about missing. What keeps me grounded is repeating thoughts like "I could miss, or I could also just FC" or to remind myself to stay focused on the main mechanics of my body. Don't lose your reading and listening, and keep your hands as loose as possible.
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

> What are your thoughts on osu!lazer?

Reviewer: we don't use "osu!lazer" as a term

```suggestion
What are your thoughts on lazer?
```

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> At first I was really apprehensive, but I've gotten a little more comfortable with osu!lazer now. I still think there are a lot of pros and cons, and the general opinion is pretty mixed. As a competitive player, I think gameplay on Lazer is starting to feel much smoother and more comfortable... only after I spent a couple days shuffling through the frame settings and the different render settings. There's also a lot of missing features, like not being able to know what skin I'm using until I'm in a map, MP links being really weird for tournaments, and not being able to hide the scoreboard when playing multiplayer, which is pretty distracting. I also think Lazer has awesome features and they're always adding cool things, however the UI feels very crowded and I'd have to often ask where things are and how to find certain settings. Totally something you can get used to as a new player fresh in to lazer, but it can be pretty overwhelming for someone like me.

Reviewer 3: ```suggestion
At first, I was really apprehensive, but I've gotten a little more comfortable with osu!lazer now. I still think there are a lot of pros and cons, and the general opinion is pretty mixed. As a competitive player, I think gameplay on lazer is starting to feel much smoother and more comfortable... only after I spent a couple days shuffling through the frame settings and the different render settings.

There's also a lot of missing features, like not being able to know what skin I'm using until I'm in a map, MP links being really weird for tournaments, and not being able to hide the scoreboard when playing multiplayer, which is pretty distracting. I also think lazer has awesome features and they're always adding cool things, however the UI feels very crowded, and I'd have to often ask where things are and how to find certain settings. Totally something you can get used to as a new player fresh in to lazer, but it can be pretty overwhelming for someone like me.
```

Maybe! I know there's been systems like that before like Maidbot which I used and was pretty cool at the time, but I'm not sure how much incentive it has to get people to grind it. There's only so many times I can play the same pools over and over again at least... but there's actually a pretty cool osu! ELO system I heard of called the osu! Performance Ranking! (please don't silence me)

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

What other games besides osu! have you played the most?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> I think the games I have with the most hours are Archeage and Smite (unfortunately). I used to be a pretty big MMO player when I was younger, starting with Maplestory. Not really grinding or anything but just chill progression and exploring. That's also why one of my favourite games of all time has been Skyrim, which I've been playing since its release! I think I played Smite heavily for about two years, again not really competitively but I liked the gameplay a lot. I have pretty much never been competitive in a game besides osu!, probably because I can't stand random people yelling at me U_U

Reviewer 3: ```suggestion
I think the games I have with the most hours are Archeage and Smite (unfortunately). I used to be a pretty big MMO player when I was younger, starting with Maplestory. Not really grinding or anything but just chill progression and exploring. That's also why one of my favourite games of all time has been Skyrim, which I've been playing since its release!

I think I played Smite heavily for about two years, again not really competitively, but I liked the gameplay a lot. I have pretty much never been competitive in a game besides osu!, probably because I can't stand random people yelling at me U_U
```

I still have some dream scores I'd like to do, like finishing all my FC's on the Because Maybe's, maybe a cheeky HDHR FC on Sound Chimera, Euphoria, and Firelight... amongst some other goals. Competitively, I'd still like an international 1v1 badge, and to continue trying and supporting new and innovative ideas, like the recent Lazer Grand Arena tournament!

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>

And as a finishing touch, do you have anything else that you'd like to mention?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>

> I hope to continue doing my best with osu! and with content creation, and I hope I can contribute more to the game as time goes on. One of my favourite contributions has been the tournament I co-host with Fulserish, dubbed "Battle of the Box", which encouraged a lot of artists in the community to contribute to tournaments, which I think had a domino effect where many more tournaments are featuring custom artwork for custom maps! If you would like to get involved in that, please consider joining the [osu! Artist Hub](https://discord.gg/KVPDmKAkjV) Discord, where there are a lot of like-minded individuals who want to be better artists and continue contributing to the game.

Reviewer 3: is there something that can be linked regarding battle of the box? like a forum thread or something

Reviewer 3: ```suggestion
I hope to continue doing my best with osu! and with content creation, and I hope I can contribute more to the game as time goes on. One of my favourite contributions has been the tournament I co-host with [Fulserish](https://osu.ppy.sh/users/14252162), dubbed "Battle of the Box", which encouraged a lot of artists in the community to contribute to tournaments, which I think had a domino effect where many more tournaments are featuring custom artwork for custom maps! If you would like to get involved in that, please consider joining the [osu! Artist Hub](https://discord.gg/KVPDmKAkjV) Discord, where there are a lot of like-minded individuals who want to be better artists and continue contributing to the game.
```

Author: Will try to find the link rq
